geoprocessing/src/migrations/geoprocessing/1620408879000-AddUniqueConstraintsToAdminRegionsTable.ts

```diff
@@ -0,0 +1,20 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class AddUniqueConstraintsToAdminRegionsTable1620408879000
+  implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`
```

Comment:
Judging from the usage, we may want to consider adding composite index:

![image](https://user-images.githubusercontent.com/3466388/117617237-8e7be280-b16c-11eb-92e5-ffa89ba46e79.png)